Assets/Scripts/Tanks/TankView.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UIElements;
+using BulletServices;
+
+namespace TankServices
+{
+    public class TankView : MonoBehaviour
+    {
+        private TankController tankController;
+        private float movement, rotation;
+        private Rigidbody rb;
+        public Transform bulletShootPoint;
+
+        private void Awake()
+        {
+            rb = GetComponent<Rigidbody>();
+        }
+
+        public void SetTankController(TankController tankControl)
+        {
+            tankController = tankControl;
+        }
+
+        private void Update()
+        {
+            Move();
+            ShootBullet();
+        }
+
+        private void FixedUpdate()
+        {
+            //tankController.Move(movement, tankController.TankModel.MovSpeed);
+            //tankController.Rotate(rotation, tankController.TankModel.RotSpeed);
+
+            Vector3 move = transform.forward * movement * tankController.tankModel.MovSpeed * Time.deltaTime;
```

Comment:
The tank movement should be done in the controller here we need only the position as it is monobehaviour .